src/doc/rustc/src/codegen-options/index.md

```diff
@@ -530,6 +530,19 @@ This also supports the feature `+crt-static` and `-crt-static` to control
 Each target and [`target-cpu`](#target-cpu) has a default set of enabled
 features.
 
+## tune-cpu
```

Comment:
Moving this option to `-Z` made it unstable and not-actually-a-codegen-option. At the very least this should be annotated as unstable.

Reply:
Annotated, but not moved yet since I don't know where unstable opts should be documented.